lib/open_api_spex/cast_parameters.ex

```diff
@@ -57,7 +57,7 @@ defmodule OpenApiSpex.CastParameters do
           Reference.resolve_parameter(ref, components.parameters)
 
         %Parameter{schema: %Reference{"$ref": "#/components/parameters/" <> _}} = param ->
-          schema = Reference.resolve_parameter(param.schema, components.parameter)
+          schema = Reference.resolve_parameter(param.schema, components.parameters)
```

Comment:
Is it common to have schemas in the `#/components/parameters/` section of the API definition? I thought it should only be used for shared parameter definitions 🤷‍♂️